config/config_test.go

```diff
@@ -94,13 +99,13 @@ config:
 	for i, test := range tests {
 		s.Run(fmt.Sprint(i), func() {
 			cfg, err := config.NewConfigFrom(test.config)
-			s.NoError(err)
+			require.NoError(s.T(), err)
```

Comment:
Cool! I wasn't familiar with that, I think we should do this change in many of our unit tests🥇